Checkout/Checkout/Source/Model/Card+Scheme.swift

```diff
@@ -21,33 +21,33 @@ extension Card {
   ///   - Visa
   public enum Scheme: String, CaseIterable, CustomStringConvertible {
     case unknown
-    case visa
     case mada
+    case visa
     case mastercard
     case maestro
-    case amex
+    case americanExpress
     case discover
-    case diners
+    case dinersClub
     case jcb
 
     public init?(rawValue: String) {
       switch rawValue.lowercased() {
```

Comment:
```suggestion
      switch rawValue
                   .lowercased()
                   .trimmingCharacters(in: .whitespacesAndNewlines) {
```

Reply:
our API always returns the scheme without leading or trailing whitespace, did you instead mean `.filter { !.$0.isWhitespace }`